lib/src/ioctl/qbuf.rs

```diff
@@ -147,7 +148,7 @@ impl<H: PlaneHandle> Default for QBuffer<H> {
 }
 
 impl<H: PlaneHandle> QBuffer<H> {
-    pub fn set_timestamp(mut self, sec: i64, usec: i64) -> Self {
+    pub fn set_timestamp(mut self, sec: time_t, usec: c_long) -> Self {
```

Comment:
Thanks for the fix! It looks like the accurate type for `usec` is `libc::suseconds_t` though, shall we use it to make sure we things won't break again down the line?

Reply:
ack, done